apis/src/main/kotlin/org/yapp/apis/auth/strategy/KakaoAuthStrategy.kt

```
@@ -1,74 +1,57 @@
-            throw AuthException(AuthErrorCode.INVALID_CREDENTIALS, "Credentials must be KakaoAuthCredentials")
-        }
-
-        val headers = HttpHeaders().apply {
-            set("Authorization", "Bearer ${credentials.accessToken}")
-            set("Content-Type", "application/x-www-form-urlencoded;charset=utf-8")
+    override fun authenticate(credentials: AuthCredentials): UserCreateInfo {
+        return try {
+            val kakaoCredentials = validateCredentials(credentials)
+            val kakaoUser = kakaoApiHelper.fetchUserInfo(kakaoCredentials.accessToken)
+            createUserInfo(kakaoUser)
+        } catch (exception: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)